config/locales/en.yml

```
@@ -1133,10 +1133,17 @@ en:
     logged_out: "Successfully logged out."
   notes_controller:
     research_notes: "Research notes"
-    thank_you_for_contribution: "Success! Thank you for contributing open research,
-      and thanks for your patience while your post is approved by <a href='/wiki/moderation'>community
-      moderators</a> and we'll email you when it is published. In the meantime, if
-      you have more to contribute, feel free to do so."
+   thank_you_for_contribution:  |
+    <h2>Success! You did it! Hooray!</h2>
```

Ah, sorry there is an error in YAML parsing starting on this line. YAML format is so strict! 😨 

Each space is meaningful and necessary. Can you try to make the spacing consistent, for example on this line:

```suggestion
     <h2>Success! You did it! Hooray!</h2>
```

See how each indent should be 2 spaces?